packages/core/admin/ee/server/src/utils/persisted-tables.ts

```diff
@@ -82,7 +84,7 @@ async function getPersistedTables({ strapi }) {
  * @param {Array<string|{ table: string; dependsOn?: Array<{ table: string;}> }>} tableNames
  * @returns {Promise<void>}
  */
-async function setPersistedTables({ strapi }, tableNames) {
+async function setPersistedTables({ strapi }: { strapi: LoadedStrapi }, tableNames: string[]) {
```

Comment:
string[] doesn't match what we have in the old JSDOC, should we define this as a type?

Reply:
true, will update it!